src/vs/workbench/contrib/terminal/test/browser/quickFixAddon.test.ts

```diff
@@ -92,54 +85,6 @@ suite('QuickFixAddon', () => {
 					strictEqual(await (getQuickFixesForCommand([], terminal, createCommand(`gt sttatus`, output, GitSimilarOutputRegex, exitCode), expectedMap, openerService, labelService)), undefined);
 				});
 			});
```

Comment:
these weren't catching the failure anyway, so fine to remove them

Reply:
we should really have smoke tests for this

Reply:
Aren't they still testing something though?

Reply:
seemingly not since they were passing when this was totally broken